fe1-web/components/QRCode.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import TextBlock from 'components/TextBlock';
+import { KeyPairStore } from 'store/stores';
+
+const QRCode = (props: IPropTypes) => {
+  const identity = KeyPairStore.getPublicKey().toString();
+  const { visibility } = props;
+
+  return (
+    // For now, we simply display a text block with the public key
+    (visibility) ? <TextBlock bold text={`ID (pk) :   ${identity}`} /> : null
+  );
```

Comment:
ternary operators are easily hard to read. Suggestion:

```suggestion
  // For now, we simply display a text block with the public key
  return (visibility)
    ? (<TextBlock bold text={`ID (pk) :   ${identity}`} />)
    : null;
```